README.md

```diff
@@ -2171,3 +2171,9 @@ My name is Kateryna and I'm happy to join this community!
 
 My name is [Dima](https://github.com/dimonlakhin). I decided to switch to a new profession. So I am glad to join the community. I hope to get your help and support.
 ![This is my own cat Shaya](./assets/images/dimon89.jpg)
+
+## Hi, everyone!
+My name is [Andrii Shchur](https://github.com/Acerpo). 
+Last time I did not start correctly, so we start from the beginning on 07/19/2022
+I want to learn frontend! 
+![Cat](./assets/images/mycat_A.jpg)
```

Comment:
As you see, Markdown palced the cat on the "same line" as your text, in Markdown you need a full empty line most blocks to consider them "on different lines". Putting an empty line before your picture will move it under the text nicely.